docs/specs/clients/notify/client-api.md

```diff
@@ -44,13 +43,16 @@ abstract class Client {
   public abstract deleteNotifyMessage(params: { id: number }): Promise<void>;
   
   // decrypt notify subscription message
-  public abstract decryptMessage(topic: string, encryptedMessage: string): Promise<NotifyMessage>;
+  public abstract decryptMessage(params: { topic: string, encryptedMessage: string }): Promise<NotifyMessage>;
   
-  // Enable Sync by registering sync keys
-  public abstract enableSync({
+  // registers a blockchain account with an identity key if not yet registered on this client
+  // if enableSync is true then register sync keys
+  // returns the public identity key.
+  public abstract register(params: {
         account: string, 
-        onSign: (message: string) => Cacao.Signature
-  }): Promise<void>;
+        onSign: (message: string) => Cacao.Signature,
+        enableSync: boolean
```

Comment:
Should we just enable this by default?

Reply:
I'm personally in favour of enabling Sync by default, and disallowing using Notify without Sync

Reply:
Agreed. It should always be enabled. No need for this flag

Reply:
Looks like we are all in agreement here. Flag removed